modules/oadp-velero-cpu-memory-requirements.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-velero-cpu-memory-requirements_{context}"]
+= Velero CPU and memory requirements based on collected data
+
+The following recommendations are based on observations of performance made in the scale and performance lab. The backup and restore resources can be impacted by the type of plugin, the amount of resources required by that backup or restore, and the respective data contained in the persistent volumes (PVs) related to those resources.
+
+
+== CPU and memory requirement for configurations
```

Comment:
In this table, some of the cells are blank. I would suggest putting None or N/A instead of leaving blank.